web-agent/app/worker.py

```diff
@@ -34,12 +34,15 @@
 
 timeout: int = 10
 
+outgoing_proxy = {}
```

Comment:
this is with respect to client.
proxy for calls going out of customer environment. ( to armorcode).